language-pages/typesetting.md

```diff
@@ -10,4 +10,7 @@ sidenav: language-side-nav
 
 - Understanding font normalization and how it affects typesetting
 - Defining type styles using font-family, font-size, and line-height
-- Other type considerations: weight, letterspacing, and measure (line-length)
\ No newline at end of file
+- Other type considerations: weight, letterspacing, and measure (line-length)
+
+![import button on tokens plugin]({{ site.baseurl }}/assets/img/type-infographic.png)
```

Comment:
So, this complicates the graphic a bit, but what you'd get in browser for the `merriweather` design token is `font-family: "Merriweather Web", "Georgia", "Cambria", "Times New Roman", "Times", serif`. While it's visually messier, I think it's important to highlight that designers should be thinking about the whole font stack, not just the preferred font (which may not always be available or be slow to load for some reason).

Reply:
Also need accurate alt text here.